### DISCUSSION TITLE
Wallet not synced, farm does'nt show plot, plotting shows 13% and errors in log

### DISCUSSION THREAD

**Discussion by CryptoPrince9** [Chia-Network/chia-blockchain#5933] [Support]
**SOS, what should I do?
Full node synced, but Wallet not synced, the farm doesn't show plot, plotting shows 13% and errors in log as follows:**
_"Only wrote 0 of 1048572 bytes at offset 49282884 to "G:\\Chia\\plot-k33-2021-05-23-08-03-71ab312a5b7e1411eedef9aa00569f884906fa2bce0018b6303bd87b05d1e620.plot.p1.t1.sort_bucket_010.tmp"with length 49283072. Error 1. Retrying in five minutes.
Only wrote 0 of 1048572 bytes at offset 49282884 to "G:\\Chia\\plot-k33-2021-05-23-08-03-71ab312a5b7e1411eedef9aa00569f884906fa2bce0018b6303bd87b05d1e620.plot.p1.t1.sort_bucket_010.tmp"with length 49283072. Error 1. Retrying in five minutes.
Only wrote 0 of 1048572 bytes at offset 49282884 to "G:\\Chia\\plot-k33-2021-05-23-08-03-71ab312a5b7e1411eedef9aa00569f884906fa2bce0018b6303bd87b05d1e620.plot.p1.t1.sort_bucket_010.tmp"with length 49283072. Error 1. Retrying in five minutes.
Only wrote 0 of 1048572 bytes at offset 49282884 to "G:\\Chia\\plot-k33-2021-05-23-08-03-71ab312a5b7e1411eedef9aa00569f884906fa2bce0018b6303bd87b05d1e620.plot.p1.t1.sort_bucket_010.tmp"with length 49283072. Error 1. Retrying in five minutes."_

**Comment by Tofandel** [comment]
Maybe read the basics?

You clearly don't have enough space on your plotting SSD so plotting is failing

You need at least 300Gb per parallel plot

**Comment by BrandtH22** [comment]
@CryptoPrince9 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
